tests/unit/lib/samconfig/test_samconfig.py

```diff
@@ -28,14 +27,25 @@ def _check_config_file(self):
         self.assertTrue(self.samconfig.sanity_check())
         self.assertEqual(SAM_CONFIG_VERSION, self.samconfig.document.get(VERSION_KEY))
 
-    def _update_samconfig(self, cmd_names, section, key, value, env):
-        self.samconfig.put(cmd_names=cmd_names, section=section, key=key, value=value, env=env)
+    def _update_samconfig(self, cmd_names, section, key, value, env=None):
+        if env:
+            self.samconfig.put(cmd_names=cmd_names, section=section, key=key, value=value, env=env)
+        else:
+            self.samconfig.put(cmd_names=cmd_names, section=section, key=key, value=value)
```

Comment:
Does this suffice?

```suggestion
        self.samconfig.put(cmd_names=cmd_names, section=section, key=key, value=value, env=env)
```

Reply:
this will pass a `env=None` to `samconfig.put` while we want to not pass a value at all so that the default value ([DEFAULT_ENV](https://github.com/aws/aws-sam-cli/blob/e4853b39bf8011132d08a12fdbb8c2e18437b9d9/samcli/lib/config/samconfig.py#L19)) is used. see [here](https://github.com/aws/aws-sam-cli/blob/e4853b39bf8011132d08a12fdbb8c2e18437b9d9/samcli/lib/config/samconfig.py#L90)

Reply:
Oh I see